### PR TITLE
Add reverse proxy tutorial for RUM On Premise

### PR DESCRIPTION
**Pull request by elenacoscale**

### CODE REVIEW

File: _docs_advanced/index.md [hunk 1]
```diff
@@ -7,5 +7,8 @@ description: Privacy and security information for the CoScale platform.
 ## [Privacy / Security]({{ site.baseurl }}/advanced/security)
 Information about how are system works and security measures we take to ensure your data stays safe and available.
 
+## [On premise]({{ site.baseurl }}/advanced/onpremise)
```
Comment (kidk) [inline _docs_advanced/index.md]: Can you change this to header, so 

```
## On premise

* [Setting up a "reverse proxy" for RUM traffic]()
```

File: _docs_advanced/onpremise.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
```
Comment (kidk) [inline _docs_advanced/onpremise.md]: Can you change the name of the file to _docs_advanced/onpremise/reverse-proxy.md

File: _docs_advanced/index.md (hunk 1 above)
Comment (elenacoscale) [reply _docs_advanced/index.md]: Done

**Comment by kidk** [comment]
There are some build errors:

```
- ./_site/advanced/index/index.html
  *  internally linking to /advanced/onpremise, which does not exist (line 98)
- ./_site/advanced/onpremise/reverse-proxy/index.html
  *  error parsing attribute name (line 7)
  *  error parsing attribute name (line 7)
```

**Pull request merged**